include/mbedtls/md.h

```diff
@@ -148,7 +148,7 @@ void mbedtls_md_init( mbedtls_md_context_t *ctx );
  *                  Calling this function if you have previously
  *                  called mbedtls_md_init() and nothing else is optional.
  *                  You must not call this function if you have not called
- *                  mbedtls_md_init().
+ *                  mbedtls_md_init(). The \p ctx must not be \c NULL.
```

Comment:
Blocker: `xxx_free()` functions should accept and silently return when when being passed `NULL` pointers.